src/fdk_rdf_parser/classes/contactpoint.py

```diff
@@ -11,3 +14,15 @@ class ContactPoint:
     organizationUnit: Optional[Dict[str, str]] = None
     hasURL: Optional[str] = None
     hasTelephone: Optional[str] = None
+
+
+@dataclass
+class CVContactPoint:
+    uri: Optional[str] = None
+    email: Optional[List[str]] = None
+    telephone: Optional[List[str]] = None
+    contactPage: Optional[List[str]] = None
+    language: Optional[List[SkosCode]] = None
+    openingHours: Optional[Dict[str, str]] = None
+    specialOpeningHours: Optional[List[OpeningHoursSpecification]] = None
+    hoursAvailable: Optional[List[OpeningHoursSpecification]] = None
```

Comment:
Foreslår vi døper om `ContactPoint` til `DcatContactPoint` slik at vi følger samme mønster for alle dataklassene og viser hvilken spek det tilhører.

Reply:
evt `DCATContactPoint`?

Reply:
d2d4c63